python/cac_tripplanner/destinations/models.py

```diff
@@ -25,7 +26,10 @@ class Destination(models.Model):
                                help_text=('The map automatically updates as the address is typed, '
                                           'but may be overridden manually if incorrect.'))
     point = models.PointField()
-    image = models.ImageField(upload_to='destinations/', null=True)
+    image = models.ImageField(upload_to='destinations/', null=True,
```

Comment:
Now that we have an AWS account set up, should we create a bucket to upload images to or is that a separate task (I'm not sure what's involved with that)